tests/test_tmjob.py

```diff
@@ -29,6 +29,8 @@
 TEST_ANGLES = TEST_DATA_DIR.joinpath('tomogram_angles.mrc')
 TEST_CUSTOM_ANGULAR_SEARCH = TEST_DATA_DIR.joinpath('custom_angular_search.txt')
 TEST_WHITENING_FILTER = TEST_DATA_DIR.joinpath('tomogram_whitening_filter.npy')
+TEST_JOB_JSON = TEST_DATA_DIR.joinpath('tomogram_job.json')
+TEST_JOB_JSON = TEST_DATA_DIR.joinpath('tomogram_job.json')
```

Comment:
double, 1 should be removed
```suggestion
TEST_JOB_JSON = TEST_DATA_DIR.joinpath('tomogram_job.json')
```